xapian-core/cluster/Makefile.mk

```diff
@@ -0,0 +1,6 @@
+EXTRA_DIST +=\
+	weight/Makefile
```

Comment:
Running `make dist` will tell you if these kinds of things work. You can always push fixup commits to a PR after opening it.